AutoLayoutTextViews/ALAutoResizingTextView.m

```diff
@@ -153,20 +153,23 @@ - (void)setNewHeightWithoutAnimation
 
 - (void (^)())animationBlock
 {
+  __weak ALAutoResizingTextView *weakSelf = self;
   return  ^{
-    if ([self.delegate respondsToSelector:@selector(textView:willChangeFromHeight:toHeight:)]) {
-      [self.delegate textView:self willChangeFromHeight:self.oldHeight toHeight:self.newHeight];
+    ALAutoResizingTextView *innerSelf = weakSelf;
```

Comment:
Change    
    `ALAutoResizingTextView *innerSelf = weakSelf;`

to
    `__strong ALAutoResizingTextView *strongSelf = weakSelf;`

per common conventions, and likewise rename other `innerSelf` calls to `strongSelf`.